src/components/CurrencyLogo/index.tsx

```diff
@@ -2,7 +2,7 @@ import { Currency, ETHER as FUSE, Token, ChainId } from '@fuseio/fuse-swap-sdk'
 import React, { useMemo } from 'react'
 import styled from 'styled-components'
 
-import FuseLogo from '../../assets/images/fuse-logo.png'
+import FuseLogo from '../../assets/svg/fuse-logo.svg'
```

Comment:
How do we use this?

Reply:
on Fusenet since fuse is the native currency we handle it differently because we can't add it to the tokenlist